src/docs/sections/testimonials.hbs

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html>
+<head>
+  {{> html-head title="Services & Trustpilot Sections" }}
+</head>
+<body>
+  <section class="pt-6 pb-6 text-center container">
+    <div class="row">
```

Comment:
updated based what we have discussed, but i'm still leave class pt-3 pb-3 inside div element didn't move it to parent element because i need padding on div for mobile.